docs/monitoring/manually-setting-jmx-containers.md

```diff
@@ -0,0 +1,67 @@
+title: Manual set up of JMX monitoring for Java process running in containers
+description: In some cases it may be required to set up JMX settings of monitored process manually
+
+There are two things which have to be done to make this happen:
+- add process arguments which tell JVM to expose JMX port with specific authentication info
```

Comment:
- add process arguments which tell JVM to expose JMX port with specific authentication info
=>
- add process arguments which tell JVM to expose a JMX port with specific authentication info